File: .tmp_py4013/batch_08_dump/2880_select_data.js
// LeetCode 2880 - Select Data
// https://leetcode.com/problems/select-data/

/**
 * @param {object[]} students
 * @return {object[]}
 */
var selectData = function(students) {
    return students
        .filter((r) => r.student_id === 101 || r[0] === 101)
        .map((r) => (Array.isArray(r) ? { name: r[1], age: r[2] } : { name: r.name, age: r.age }));
};

File: .tmp_rs4013/rust_check_12/2880_select_data.rs
struct Solution;
// LeetCode 2880 - Select Data
// https://leetcode.com/problems/select-data/

impl Solution {
    pub fn select_data(students: Vec<Vec<i32>>) -> Vec<Vec<i32>> {
        students
            .into_iter()
            .filter(|r| !r.is_empty() && r[0] == 101)
            .map(|r| r[1..].to_vec())
            .collect()
    }
}

fn main() {}
